README.md

```diff
@@ -58,3 +58,32 @@ import FLAnimatedImage from 'react-native-flanimatedimage';
   <FLAnimatedImage style={style} source={source} onLoadEnd={this.onLoadEnd} />
 ...
 ```
+
+## Android
+
+Android uses regular images, but you can still call the component
+
+```js
+import FLAnimatedImage from 'react-native-flanimatedimage';
+
+  <FLAnimatedImage style={style} source={source} onLoadEnd={this.onLoadEnd} />
```

Comment:
I remember `onLoadEnd` doesn't return the size of the image for official `Image` component, so I think it would be more proper  to tell that on Android we simply export the `Image` from React Native